packages/formation/sass/base/_b-mixins.scss

```diff
@@ -87,19 +87,13 @@
   }
 }
 
-@mixin focus-gold-light-outline($offset: 2) {
+@mixin focus-gold-vivid-outline($offset: 2) {
   outline: $focus-outline;
   outline-offset: #{$offset}px;
 }
 
-@mixin focus-gold-lighter-highlight {
-  background: $color-gold-lighter;
-  outline: 2px solid $color-gold-lighter;
-  outline-offset: 0;
-}
```

Comment:
I removed this because it was related and it didn't seem used. I saw it was added in https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/commit/1b95985df123ef0ffb5698e8f1c5021c620fd702, but I couldn't find a PR for it. I also did two different `git bisect`s in `vets-website` so I could try to pin down:

1. Where the `formation` version changed to `3.0.0`
    - I don't think this was useful because we don't have any release notes tied to versions - I was just going off of the tag on the commit
1. Where `focus-gold-lighter-highlight` was used in `vets-website`.

I didn't find it anywhere but I may have taken a wrong direction in a bisect and missed it, especially if it was around for a short amount of time.